### DISCUSSION TITLE
NO_COLOR=1 background opacity

### DISCUSSION THREAD

**Discussion by chapmanjacobd** [Textualize/textual#2830] [Ideas]
It might make sense for `NO_COLOR=1` to not override the terminal background (default color), or use a simpler rendering mode. For reference, here is how `ncdu` does it:

![ksnip_20230623-180852](https://github.com/Textualize/textual/assets/7908073/3ffba297-8890-44d0-9efe-b11a7d0a70d1)

Without `NO_COLOR=1`:

![ksnip_20230623-180816](https://github.com/Textualize/textual/assets/7908073/f17b7713-9dbe-4111-87f8-febcb51bf750)

This gets part of the way there but it probably also breaks some things:

```py
class DiskUsage(App):
    CSS ='''
    Screen {
        opacity: 0%;
    }

    Label {
        text-style: reverse;
    }
    '''
```

**Comment by willmcgugan** [comment]
I'm not sure what "transparent" would mean in this context. But [NO_COLOR](https://no-color.org/) has a well defined meaning, which doesn't offer much flexibility.
